mesheryctl/internal/cli/root/system/token.go

```diff
@@ -0,0 +1,170 @@
+package system
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var ctx string
+var tokenCmd = &cobra.Command{
+	Use:   "token",
+	Short: "Manage Meshery user tokens",
+	Long: `
+	Manipulate user tokens and their context assignments in your meshconfig`,
+	Args: cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
+			return errors.New(utils.SystemError(fmt.Sprintf("invalid command: \"%s\"", args[0])))
+		}
+		return nil
+	},
+}
+
+var addTokenCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add a token to meshery config",
+	Example: `
+	mesheryctl system token add <token-name> -f <token-path>
+	mesheryctl system token add <token-name> (default path is auth.json)
+	`,
+	Args: cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		tokenName := args[0]
+		if tokenPath == "" {
+			tokenPath = "auth.json"
+		}
+
+		token := config.Token{
+			Name:     tokenName,
+			Location: tokenPath,
+		}
+		if err := config.AddTokenToConfig(token, utils.DefaultConfigPath); err != nil {
+			return errors.Wrap(err, "Could not add specified token to config")
+		}
+		log.Printf("Token %s added successfully!", tokenName)
+		return nil
+	},
+}
+var deleteTokenCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete a token from meshery config",
+	Example: `
```

Comment:
Where is the long description?

Reply:
Example and Short description pretty much summed up the command so didn't add Long. But to stick with convention, will add for sure now.